### ISSUE TITLE
Sass interpolation syntax don't work. How to fix it?

### ISSUE THREAD

**Issue by oscarhandsome** [MohammadYounes/rtlcss#131]
Hi @MohammadYounes and developers! Thanks for great job! But i have an issue, please help me, if you can resolve it.

i'm using simple **gulp, sass, jade** in my work. I added [rtlcss  ](https://github.com/MohammadYounes/rtlcss) and [gulp-rtlcss](https://github.com/jjlharrison/gulp-rtlcss)
i took example with SASS interpolation from this [page](https://rtlcss.com/learn/usage-guide/value-directives/#tip)

i wrote in my scss: 

`.font-19 {
   font-family: 'FF-Shamel-Medium';
   font-size: 17px #{"/*rtl:5px*/"};
 }`

my [gulpfile here](https://runkit.com/oscarhandsome/gulpfile-js)

at first i compilate SASS, then jade, etc.

for Mohammad's rtlcss how i understand i can use CLI, or took css, then create new css-rtl version.

i'm doing this with 

`gulp.task('rtl', function () {
   return gulp.src('src/assets/css/main.css')
       .pipe(rtlcss())
       .pipe(rename({ suffix: '-rtl' })) // Append "-rtl" to the filename.
       .pipe(sourcemaps.write()) // Output source maps.
       .pipe(gulp.dest('src/assets/css'))
});`
i turn on this task when my css created. This is workink fine, but interpolation is loosing.
but i'm not understand what wrong with SASS, why interpolation not work? Should i change my **sass task** in gulpfile?

[image ](http://almall.elbargo.ru/interpolation.jpg) of the example page

**Comment by oscarhandsome** [comment]
i hide this lines in my gulp file, interpolation is working correct, but i don't still understand where is an issue

```
// .pipe(autoprefixer({
//     browsers: ['last 3 versions'],
//     cascade: false
// }))
// .pipe(cleancss({compability: 'ie8'}))
// .pipe(remember('styles'))
// .pipe(gulpIf(isDevelopment, sourcemaps.write()))
```

**Comment by MohammadYounes** [comment]
Try using Special/Important comments `/*!*/`:

```scss
.font-19 { font-family: 'FF-Shamel-Medium'; font-size: 17px #{"/*!rtl:5px*/"}; }
```

**Comment by oscarhandsome** [comment]
@MohammadYounes 
Yeah! Thank worked! [image ](http://almall.elbargo.ru/interpolation2.jpg) Thanks a lot!
But how i know !important not good practice in CSS rules. Do yo uthink this good practice?

anyway, i will support you project, great job!
